tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "baseUrl": ".",
     "noImplicitAny": false,
     "allowJs": false,
+    "strictNullChecks": false,
```

Comment:
Please enforce strict mode everywhere possible. Including this option and `noImplicitAny` or else this codebase is no different from JavaScript and is very easy to break.

Reply:
resolved